.github/workflows/request-reviews.yml

```diff
@@ -0,0 +1,24 @@
+name: Request reviews for PRs
+
+on:
+  pull_request_target:
+    types: [opened]
```

Comment:
I am -1 for this approach. If the PR is opened and does not touch the deps folder but after opening another commit changes the deps folder, than it wont run.

Reply:
I can make a separate workflow specific to the deps folder, but this is more for an overall migration